CenterForOpenScience/ember-osf-web
# [ENG-1385]Metadata in review page

- Ticket: [ENG-1385]
- Feature flag: n/a

## Purpose
- Display the Draft Metadata information in the Review page of Registries submission

## Summary of Changes
- Add new component `ReviewMetadataRenderer` to render Metadata info
- Add `LabelRenderer` within `ReviewMetadataRenderer` to render heading and validation error for given field
- Made small changes to existing review rendering templates to accommodate changes needed for `ReviewMetadataRenderer`
- Add css rules to `ReviewMetadataRenderer` and `LabelRenderer` to match the style present in `ReviewFormRenderer`
- Borrowed and slightly modified `tag-manager` component from @adlius's [tags field PR](https://github.com/CenterForOpenScience/ember-osf-web/pull/918)
- Added a `schemaBlocks` property to the `DraftRegistrationManager` for the `NavigationDropdown` widget

## Screen Shots
- Metadata section:
<img width="688" alt="Screen Shot 2020-02-20 at 3 05 34 PM" src="https://user-images.githubusercontent.com/51409893/74973997-7480a380-53f2-11ea-94fc-ae16a0c6f7b5.png">

- Schema form section:
<img width="442" alt="Screen Shot 2020-02-20 at 3 03 44 PM" src="https://user-images.githubusercontent.com/51409893/74973831-34212580-53f2-11ea-90ef-6859e5148561.png">

## Side Effects
- Some existing managers/components from Editable Metadata were changed, but functionality should remain the same for those.

## QA Notes
- This PR should add the Metadata info to the review page, so please check to make sure that any changes users make to the Metadata is reflected there, and that any validation errors are showing up as appropriate.
- As mentioned earlier, the Editable Metadata components were touched on, but those functionalities should remain unchanged


[ENG-1385]: https://openscience.atlassian.net/browse/ENG-1385
